packages/cqrs/lib/cqrs.dart

```diff
@@ -28,6 +28,13 @@
 /// // Fetching first page of flowers
 /// final flowers = await cqrs.get(AllFlowers(page: 1));
 ///
+/// // Handling query result
+/// if (result.isSuccesful) {

```

Comment:
successful is an adjective, failure is noun. please use one

Reply:
Fixed it.